### DISCUSSION TITLE
Does Magick.NET comes with Ghostscript?

### DISCUSSION THREAD

**Discussion by tomasr78** [dlemstra/Magick.NET#1023] [Help] [answered]
I can turn Images to PDF without installing GS and setting up `MagickNET.SetGhostscriptDirectory(@"C:\Magick.NET\GS");`
How can it be? Does Magick.NET come with GS?

Fully working code to turn image to pdf. 
```
           using (var imageCollection = new MagickImageCollection())
            {
                imageCollection.Add(MagickImage(@"C:\jun_pdf_image.png"));
                imageCollection.Add(MagickImage(@"C:\original.jpg"));
                
                
                imageCollection.Write(@"C:\Projects\_temp\output.pdf");
            }
```

**Comment by dlemstra** [comment, marked as answer]
We don't need Ghostscript to write PDF files. The ImageMagick library only requires it to read PDF files.
